nemo/collections/tts/models/radtts.py

```diff
@@ -443,12 +442,13 @@ def input_example(self, max_batch=1, max_dim=400):
         par = next(self.parameters())
         sz = (max_batch, max_dim)
         # sz = (max_batch * max_dim,)
-        inp = torch.randint(0, 94, sz, device=par.device, dtype=torch.int64)
+        # Pick up only pronouncible tokens
+        inp = torch.randint(32, 64, sz, device=par.device, dtype=torch.int64)
```

Comment:
Why is this needed?

Reply:
This was my attempt to make sure there are no entries of zero length generated. LSTM cannot handle 0-length elements in input. That did not help 100% percent - also there was f0_pad that would add zeroes - so I had to modify clip() to ensure non-zero lengths are generated

Reply:
I don't understand. The LSTMs will see either the text_length input, or the spectrogram_length input. For text, there will be no 0-length elements since each character has length 1. When expanded from text_len to spectrogram_length, there can be 0-length elements but those characters wouldn't exist in the expanded tensor. I don't see why the LSTM will ever see a 0-length element

Reply:
Well, I definitely was encountering 0s - printed them out and observed by myself. Has to modify clamp() call for that. If that is not something expected to happen, we should investigate.

Reply:
Ok, will take a look after this PR